doc/man/8/ceph-osd.rst

```diff
@@ -18,14 +18,16 @@ Synopsis
 Description
 ===========
 
-**ceph-osd** is the object storage daemon for the Ceph distributed file
-system. It is responsible for storing objects on a local file system
-and providing access to them over the network.
-
-The datapath argument should be a directory on a xfs file system
-where the object data resides. The journal is optional, and is only
-useful performance-wise when it resides on a different disk than
-datapath with low latency (ideally, an NVRAM device).
+**ceph-osd** is the **o**\bject **s**\torage **d**\aemon for the Ceph
+distributed file system. It manages data on local storage with redundancy and
+provides access to that data over the network. 
+
+For FileStore-backed clusters, the argument of the ``--osd-data datapath``
```

Comment:
Should we drop mention of Filestore?  (lowercase s)

Reply:
We probably will, but I want to do that in one fell swoop (not here) (and probably one fell swoop made up of about two-hundred pull requests, so not much of a fell swoop at all, really).